### ISSUE TITLE
[tooltip/overlay] Unable to use aria-labelledby

### ISSUE THREAD

**Issue by erikkroes** [ing-bank/lion#269]
Currently, the localOverlayController automatically sets an `aria-describedby` on the invoker, with a reference to the content. When you use the tooltip with an ℹ️icon, and provide a tooltip with additional information, `aria-describedby` is the appropriate relationship. 

You may want to use a tooltip as a visual way to show a label or accessible name. For example, when you have a button with only an icon as content. In that case you would want to set `aria-labelledby` instead.

And when you use a local overlay for something like a menu, you don't want either `aria-describedby` or `aria-labelledby`.

I suggest making the `aria-describedby` in localOverlay to be optional, with another option to set an `aria-labelledby` instead. Suggested API in #267

**Comment by tlouisse** [comment]
In the new OverlayController we have an `isTooltip` option and there is a TODO for adding this.
https://github.com/ing-bank/lion/blob/master/packages/overlays/src/OverlayController.js#L216

We can change this line to `this.invokerNode.setAttribute(this.invokerRelation === 'label' ? 'aria-labelledby' : 'aria-describedby', this._contentId);`
